ee/clickhouse/views/person.py

```diff
@@ -102,7 +103,12 @@ def calculate_path_persons(self, request: Request) -> Dict[str, Tuple[list, Opti
         team = self.team
         filter = PathFilter(request=request, data={"insight": INSIGHT_PATHS})
 
-        people, should_paginate = ClickhousePathsPersons(filter, team).run()
+        funnel_filter = None
+        funnel_filter_data = request.GET.get("funnel_filter")
```

Comment:
Note for both Paths and Paths Person endpoint: We allow both POST and GET requests, and I think this would bork for POST requests.

Actually, I think funnels moved to POST because URL too long was becoming a problem. For a Paths request with additional funnel data, this is even more prevalent. Perhaps makes sense to only allow POST? Simplifies cases as we wouldn't have to support both.

Reply:
Will need to update this now too, since you're POSTing the data?

Reply:
yeah